### ISSUE TITLE
compiler options are disappeared on github actions

### ISSUE THREAD

**Issue by redboltz** [redboltz/mqtt_cpp#738]
I set -Wall -Wextra -Werror etc but they are disappeared.
I'm investigating it on #737 

If someone send a new PR, I don't merge before the issue would be fixed.
And then, I would ask the contributor to rebase the PR.

**Comment by redboltz** [comment]
Fixed.
